### PR TITLE
Add delay on process send message telegram

### PR DESCRIPTION
**Pull request by ayocodingit**
### Overview
- change name network docker-compose
- add delay 3 seconds
- add generate File Path

## Issue limit bot telegram
image

## Solution 
with the limit on the telegram API, the solution is to add a delay when executing queues, seen in the image below, there is a 3 seconds delay.
image

cc: @jabardigitalservice/jds-backend 

### Evidence
title: Menambahkan delay pada saat pengiriman data ke telegram IT Dev Evidence
project: Digiteam
participants: @firmanalamsyah580 @abdurrahman_adianto @fimz17 @sandisunandar

### CODE REVIEW

**Comment by azophy** [comment]
the mentioned Telegram Docs said: "the API would not allow more than 30 messages *per second*", so I think rather than 60 seconds delay, 1 second should be enough. Waiting for 60 second before the message finally being sent is too long in my oppinion. CMIIW

**Comment by ayocodingit** [comment]
> the mentioned Telegram Docs said: "the API would not allow more than 30 messages _per second_", so I think rather than 60 seconds delay, 1 second should be enough. Waiting for 60 second before the message finally being sent is too long in my oppinion. CMIIW

very good opinion
I also think, that waiting for 1 minute is too long, 
thank you very much, 
I will change the delay to 1 second :+1: :partying_face:

**Comment by ayocodingit** [comment]
> the mentioned Telegram Docs said: "the API would not allow more than 30 messages _per second_", so I think rather than 60 seconds delay, 1 second should be enough. Waiting for 60 second before the message finally being sent is too long in my oppinion. CMIIW

oh yeah @azophy  i just remembered, why i put 1 minute because there is a limit from bots can't send more than 20 messages in the same group, maybe i set 20 seconds to prevent bot issues

File: utils/execJob.js [hunk 1]
```diff
@@ -1,8 +1,10 @@
 import sendTelegram from './sendTelegram.js'
 import templateBody from './templateBody.js'
+import delay from './delay.js'
 
 const execJob = async (job, done, payload) => {
   try {
+    await delay();
```
Comment (firmanJS) [inline utils/execJob.js:7]: kenapa tidak menggunakan delay dari bull ?

File: server.js [hunk 2]
```diff
@@ -20,10 +20,7 @@ app.post('/webhook/:secret/:git', async (req, res) => {
     const git = req.params.git
     if (!isMerged(git, req.body)) return res.send('pending ...')
     const queue = connectQueue(git)
-    queue.add({ git: git, body: req.body }, {
-      delay: 60000,
-      attempts: 2
-    })
+    queue.add({ git: git, body: req.body })
```
Comment (firmanJS) [inline server.js:23]: Ini bisa di buat simple
const body = req.body

queue.add({git, body})

**Review by firmanJS — CHANGES_REQUESTED** [review]

File: utils/execJob.js (hunk 1 above)
Comment (ayocodingit) [reply utils/execJob.js:7]: delay from bull, only delays the execution time, but it takes time interval between processes :smile: 
the issue is giving delay time when sending telegrams

File: utils/screenshot.js [hunk 3]
```diff
@@ -19,9 +22,17 @@ const options = {
...
+    fs.mkdirSync(dir)
+  }
+
+  return `${dir}/${Date.now()}${Math.random()}.png`
+}
+
 const screenshot = async (url, git) => {
   const option = options[git]
-  let filePath = `tmp/${Date.now()}${Math.random()}.png`
+  let filePath = generateFilePath()
```
Comment (firmanJS) [inline utils/screenshot.js:35]: Kenapa tidak menggunakan const ?

File: server.js (hunk 2 above)
Comment (ayocodingit) [reply server.js:23]: nice :+1:

File: utils/screenshot.js (hunk 3 above)
Comment (ayocodingit) [reply utils/screenshot.js:35]: because filePath will be set to null if page is not equal to url :smiley:

**Review by firmanJS — APPROVED** [review]

**Comment by firmanJS** [comment]
Gass LGTM !

**Comment by sandisunandar99** [comment]
sepertinya sudah cukup ya review nya,, L.G.T.M 🚀 👍

**Review by sandisunandar99 — APPROVED** [review]

**Review by rachadiannovansyah — APPROVED** [review]
lgtm!

**Pull request merged**